packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardPreview.types.ts

```diff
@@ -43,6 +43,11 @@ export interface IDocumentCardPreviewProps extends IBaseProps<{}> {
    * Optional override class name
    */
   className?: string;
+
+  /**
+   * Minimum number of documents to display before adding the overflow text
```

Comment:
this should be 'Maximum number of documents' right? And '@defaultvalue 3' to the comment?